WeakAuras/WeakAuras.lua

```diff
@@ -3143,6 +3142,74 @@ function WeakAuras.CancelAnimation(region, resetPos, resetAlpha, resetScale, res
   end
 end
 
+function WeakAuras.GenerateDataStub(regionType, id)
+  local stub = {
+    id = id,
+    regionType = regionType,
+    activeTriggerMode = WeakAuras.trigger_modes.first_active,
+    disjunctive = "all",
+    trigger = {
+      type = "aura",
+      subeventPrefix = "SPELL",
+      subeventSuffix = "_CAST_START",
+      debuffType = "HELPFUL",
+      names = {},
+      spellIds = {},
+      event = "Health",
+      unit = "player"
+    },
+    untrigger = {},
+    load = {
```

Comment:
Do we need all of that in the DataStub? Keeping that up to date with other places is unlikely to happen.

Reply:
The stub should be be absolute minimum data which is common to all auras. I’ll admit I was a bit overzealous when writing it, mostly to stamp down on false positives in WeakAuras.diff. I’ll see how much I can trim it down without adversely affecting diff.